stdlib/asyncio/sslproto.pyi

```diff
@@ -66,7 +66,10 @@ class _SSLProtocolTransport(transports._FlowControlMixin, transports.Transport):
     _sendfile_compatible: ClassVar[constants._SendfileMode]
 
     _loop: events.AbstractEventLoop
-    _ssl_protocol: SSLProtocol
+    if sys.version_info >= (3, 11, 1):
```

Comment:
```suggestion
    if sys.version_info >= (3, 11):
```
We don't do patch releases on typeshed, we pretend everything is as it was in the most recent release of the series.